css/index.css

```diff
@@ -164,12 +175,177 @@ header .logo {
+    .container {
+        width: 100%;
+        padding-left: 2.2%;
+        padding-right: 2.2%;
+    }
+    header {
+        margin: 1rem;
+        width: 100%;
+    }
+
+    header nav{
+        width: 100%;
+    }
+
+    .cta {
+        width:100%;
+    }
+
+    .cta img {
+        width: 50%;
+        height: 50%;
+    }
+
+    .cta .cta-text {
+        margin: 20px;
+        margin-top: 7rem;
+        margin-bottom: 0;
+        font-size: 6.9rem;
+    }
+
+    .cta .cta-text button {
+        height: 30px;
+        width: 55%;
+        margin: 8rem;
+    }
+   
+    .main-content, .bottom-content, .top-content {
+        width: 100%;
+    }
+
+    .main-content img {
+        width: 100%
+    }
+}
+
+@media (max-width: 700px) {
+    .cta {
+        flex-wrap: wrap;
+        justify-content: center;
+    }
+    header {
+        flex-wrap: wrap;
+        justify-content: center;
+    }
+    header nav {
+        flex-wrap: wrap;
+        width:100%;
+    }
+    header .logo {
+        margin: 0;
+        order: -1;
+        align-self: center;
+        margin-top: 10px;
+    }
+}
+
+@media (max-width: 400px) {
+    .container {
+        width: 100%;
+        padding-left: 0;
+        padding-right: 0;
+        /* display: flex;
+        flex-direction: column;
+        justify-content: center; */
+    }
+    .container header {
+        justify-content: center;
+        flex-direction: column;
+        align-items: center;
+        width: 100%;
+        margin: 0;
+    }
+    header .logo {
+        margin: 0;
+        order: -1;
+        align-self: center;
+        margin-top: 20px;
+        margin-bottom: 15px;
+    }
+
+    .container header nav {
+        flex-direction: column;
+        justify-content: center;
+        width: 100%;
+        margin:0;
+        padding: 0;
+        align-items: center;
+    }
+
+    header nav a {
+        margin:0;
+        font-size: 2.3rem;
+        width: 100%;
+        padding: 19px;
+        text-align: center;
+        border-bottom: solid black 1px;
+    }
+
+    header nav a:first-child {
+        border-top: solid black 1px;
+    }
+
+    .cta {
+        width:100%;
+        flex-direction: column;
+        align-items: center;
+    }
+
+    .cta .cta-text {
+        margin-top: 20px;
+    }
+    .cta .cta-text button {
+        margin-top: 30px;
+        margin-bottom: 30px;
+        width:54%
+    }
+
+    .cta img {
+        width: 85%;
+        padding-bottom: 13px;
+    }
+   
+    .main-content, .bottom-content, .top-content {
+        display: flex;
+        flex-direction: column;
+        width: 90%;
+        align-items: center;
+        justify-content: center;
+        margin-right: 0;
+    }
+
+    .main-content .top-content{
+        align-items: center;
+        padding-right: 0;
+        margin-right: 0;
+    }
+    .contact{
+        font-size: 1.6rem;
+        padding-left: 30px;
+    }
+    .text-content {
+        padding:10px;
+    }
+
+    .main-content .middle-img{
```

Comment:
For some reason, even though you have done an align, this image is set on the left. See what happens when you implement a margin-left here. And this goes for the horizontal lines as well 👍 Overall, really solid job on this project! I feel as if your confidence has grown so much from just this week alone.